resources/knative-serving/templates/pod-monitor-ksvc.yaml

```diff
@@ -7,8 +7,6 @@ metadata:
 {{ toYaml .Values.knative_serving.podmonitor.labels | indent 4 }}
 spec:
   podMetricsEndpoints:
-  - port: queue-metrics
-    interval: 3s
```

Comment:
how is this related to the image bump?

Reply:
Once https://github.com/kyma-project/kyma/pull/7349/files is merged, prometheus will not be able to scrape the port `queue-metrics`. Additionally, it should not be scraped by Prometheus as per this [comment](https://github.com/knative/serving/issues/4638#issuecomment-509445150).